src/triangulation.rs

```diff
@@ -322,6 +322,21 @@ pub trait Triangulation: Default {
         self.s().fixed_vertices()
     }
 
+    /// Get a vertex by its index
+    ///
+    /// To get the handle, wrap your local index in a FixedVertexHandle:
+    /// ```
+    /// let handle = FixedVertexHandle::from_index(my_index)
```

Comment:
Oh, i was afraid that might happen - `cargo test` (which runs on CI) will interpret this as a doc test and attempt to compile this.

You will either need to make this a full example like this:

```
let my_index: usize = 42;
let handle = FixedVertexHandle::from_index(my_index);
```

or use single quotes instead.

You can check if everyting works by running `cargo test --doc`

Reply:
Haha, sorry my first time writing rust doc with these code examples. Very cool that it compiles the docs too!

Sorted 👍